components/conversations/CommentForm.js

```diff
@@ -69,6 +69,28 @@ const prepareCommentParams = (values, conversationId, expenseId) => {
   return comment;
 };
 
+const initialFormState = {
+  html: '',
+  validationErrors: {},
+  isSubmitting: false,
```

Comment:
You don't need this, `useMutation` returns a `loading` param that will be `true` while submitting.